{"name": "test_register_with_mandatory_field", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <method-wrapper '__eq__' of str object at 0x000001CBE7921660>('Register Account')\n +    where <method-wrapper '__eq__' of str object at 0x000001CBE7921660> = 'Your Account Has Been Created!'.__eq__\n +    and   'Register Account' = retrived_account_creation_message()\n +      where retrived_account_creation_message = <pages.AccountSucessPage.AccountSucessPage object at 0x000001CBE7925160>.retrived_account_creation_message", "trace": "self = <test.test_register.TestRegister object at 0x000001CBE78C5BD0>\n\n    def test_register_with_mandatory_field(self):\n        home_page=HomePage(self.driver)\n        home_page.click_on_my_account_drop_menu()\n        register_page=home_page.select_register_option()\n        register_page.enter_first_name(\"L1\")\n        register_page.enter_last_name(\"L1\")\n        register_page.enter_email(\"s25s@gmail.com\")\n        register_page.enter_telephone(\"3546754873\")\n        register_page.enter_password(\"1234\")\n        register_page.enter_confirm_password(\"1234\")\n        register_page.select_radio_button()\n        time.sleep(3)\n        register_page.select_privacy_policy_button()\n        time.sleep(3)\n        account_success=register_page.click_on_continue_button()\n        time.sleep(3)\n        expected_text=\"Your Account Has Been Created!\"\n>       assert expected_text.__eq__(account_success.retrived_account_creation_message())\nE       AssertionError: assert False\nE        +  where False = <method-wrapper '__eq__' of str object at 0x000001CBE7921660>('Register Account')\nE        +    where <method-wrapper '__eq__' of str object at 0x000001CBE7921660> = 'Your Account Has Been Created!'.__eq__\nE        +    and   'Register Account' = retrived_account_creation_message()\nE        +      where retrived_account_creation_message = <pages.AccountSucessPage.AccountSucessPage object at 0x000001CBE7925160>.retrived_account_creation_message\n\ntest\\test_register.py:33: AssertionError"}, "attachments": [{"name": "stdout", "source": "69b81804-f637-4522-8f43-9e2024420d20-attachment.txt", "type": "text/plain"}], "start": 1733160343866, "stop": 1733160357832, "uuid": "1fff29eb-bb65-4dbb-9ac6-601f93ce14dc", "historyId": "f83163e43a79a58cbfd4dae2cb4bcd92", "testCaseId": "f83163e43a79a58cbfd4dae2cb4bcd92", "fullName": "test.test_register.TestRegister#test_register_with_mandatory_field", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_register"}, {"name": "subSuite", "value": "TestRegister"}, {"name": "host", "value": "Agni"}, {"name": "thread", "value": "7036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_register"}]}